mgmt_cli_test.py

```diff
@@ -152,7 +167,7 @@ def verify_backup_success(self, mgr_cluster, backup_task, keyspace_name='keyspac
     def _generate_load(self):
         self.log.info('Starting c-s write workload for 1m')
         stress_cmd = self.params.get('stress_cmd')
-        stress_thread = self.run_stress_thread(stress_cmd=stress_cmd, duration=5)
+        stress_thread = self.run_stress_thread(stress_cmd=stress_cmd)
```

Comment:
is the above message still correct: 'Starting c-s write workload for 1m' ?

Reply:
Correct! I missed that. Fixed,